src/strongbox/core.clj

```diff
@@ -305,14 +309,22 @@
    (when addon-dir
      (-> addon-dir addon-dir-map :game-track))))
 
-(defn-spec get-lenient-game-track ::sp/lenient-game-track
-  "returns the lenient/compound version of the currently selected game track. 
-  if `:retail` then `:retail-classic`, etc"
+(defn-spec get-game-track-strictness ::sp/strict?
   []
-  (case (get-game-track)
-    :classic-retail :classic-retail
-    :classic :classic-retail
-    :retail-classic))
+  (get (addon-dir-map) :strict? true))
+
+(defn-spec set-game-track-strictness! nil?
```

Comment:
this is almost identical with `set-game-track!`. Add a floating `todo` for now for our future selves and a docstr and tests